src/structures/Connection.js

```diff
@@ -184,7 +184,7 @@ export default class extends EventEmitter {
 				connectionID: this.id,
 			}, 'info')
 
-			this.send('PING')
+			this.send('PING :tmi.twitch.tv')
```

Comment:
Instead of hardcoding the host, let's use the `HOST` variable (it's deconstructed from `process.env` on line 32).